thirdparty/raknet/CMakeLists.txt

```diff
@@ -80,6 +80,8 @@ target_compile_options(raknet PRIVATE
      $<$<CXX_COMPILER_ID:MSVC>:
         /w>)
 
+set_property(TARGET raknet PROPERTY CXX_STANDARD 17)
```

Comment:
This is necessary as newer C++ stds forbid the RakString conversion of a const char[1] to a char*.